src/app_websocket.rs

```diff
@@ -340,18 +339,6 @@ impl AppWebsocket {
         }
     }
 
-    pub async fn network_info(
```

Comment:
Can the two new calls be included here? network info was replaced by network stats and network metrics on the app interface